src/stories/standalone/ImageBoxComponents/ImageController.story.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useCallback } from "react";
+import { boolean, text } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
+import "../../../i18n";
+import ImageController, {
+	ImageControllerEntry,
+} from "../../../standalone/ImageBoxControl/ImageController";
+
+export const ImageControllerStory = (): React.ReactElement => {
+	const [uploadedImages, setUploadedImages] = useState<ImageControllerEntry[]>(
+		[]
+	);
+	const infoText = text(
+		"Info Text",
+		"<ul><li>Click on the icon or drag and drop your picture Drag&Drop your image onto the grey field...</li><li>Select a gallery picture. This will then be displayed in all lists and overviews.</li><li>Your pictures look particularly good when...</li></ul>"
+	);
+	const handleChangeAction = useCallback(
```

Comment:
why are there two change handlers for the same variable? => https://github.com/Components-care/react-components/pull/33#discussion_r573532935